node_modules/cli-table3/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "cli-table3",
-  "version": "0.6.1",
+  "version": "0.6.2",
```

Comment:
I've been looking out for this line because when I published cli-table3 (using [np](https://www.npmjs.com/package/np)), it somehow didn't capture the version bump to `0.6.2` in `package.json` in the `v.0.6.2` tag. It seems this package must autocorrect this and I wanted to drop in to give my 👍 

![image](https://user-images.githubusercontent.com/260874/164372478-f240e093-ac0b-4240-81c3-367b2ccd10b8.png)

Reply:
for reference, in the future if you want to validate something similar in a published package, you can use `npm diff`:

```sh
$ npm diff --diff=cli-table3@0.6.1 --diff=cli-table3@0.6.2 package.json
```

will give you the diff (for the `package.json` file) between these two published versions of the package from the registry